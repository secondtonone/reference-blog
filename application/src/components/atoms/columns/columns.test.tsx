import renderer from 'react-test-renderer';
import Columns from '.';

describe('Match snapshots Columns', () => {
  it('with column', () => {
    const tree = renderer.create(<Columns><div className="column" /></Columns>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
