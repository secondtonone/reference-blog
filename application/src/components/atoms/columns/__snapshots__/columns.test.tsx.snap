// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshots Columns with column 1`] = `
<div
  className="box-adaptive__BoxStyled-sc-g2wy39-0 gPmSgV"
  data-test="columns"
>
  <div
    className="column"
  />
</div>
`;
